kerchunk/xarray_backend.py

```diff
@@ -0,0 +1,31 @@
+from xarray.backends import BackendEntrypoint
+import xarray as xr
+import os
+from fsspec.implementations.reference import ReferenceFileSystem
+
+
+class KerchunkBackend(BackendEntrypoint):
+    def open_dataset(
+        self,
+        filename_or_obj,
+        **kwargs,
+    ):
```

Comment:
Should there be a docstring?

Reply:
We currently don't have guidelines on how to best document entrypoints. It would be good to figure those out (and maybe standardize in a sphinx extension), but for now you will most likely only see the docstring when reading the code or when inspecting the entrypoint before directly passing it to `xr.open_dataset`